app/policies/case/sar_policy.rb

```diff
@@ -14,17 +8,29 @@ def show?
       check(:responding_team_is_linked_to_case)
   end
 
+  def new_case_link?
+    clear_failed_checks
+    check_can_trigger_event(:link_a_case) &&
+      check_user_is_a_manager_for_case
+  end
+
+  def can_close_case?
+    clear_failed_checks
+    self.case.drafting? &&
+        user.responding_teams.include?(self.case.responding_team)
+  end
+
+  def can_add_attachment_to_flagged_and_unflagged_cases?
+    false
+  end
+
   check :responding_team_is_linked_to_case do
     self.case.linked_cases.detect do |kase|
       kase.responding_team.in? user.responding_teams
     end
   end
 
-
-  def new_case_link?
-    clear_failed_checks
-    check_can_trigger_event(:link_a_case) &&
-      check_user_is_a_manager_for_case
+  check :user_is_a_responder_for_case do
```

Comment:
Should this be different from the [same check in `Case::BasePolicy`](https://github.com/ministryofjustice/correspondence_tool_staff/blob/1d3bc1540b92f5a6c17850afffbfd12d4c6251d8/app/policies/case/base_policy.rb#L465)?